File: JPAFoodFinder/src/main/java/com/skilldistillery/foodfinder/entities/Service.java
package com.skilldistillery.foodfinder.entities;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Service {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	@Column(name="service_name")
	private String serviceName;
	private String description;
	// map to service_location many to many

}

File: ngFoodfinder/src/app/models/service.ts
export class Service {

  id : number;
  serviceName : string;
  description : string;
  serviceLocations : object[] | null;

  constructor(
    id : number = 0,
    serviceName : string = '',
    description : string = '',
    serviceLocations : object[] | null = []
  ) {
    this.id = id;
    this.serviceName = serviceName;
    this.description = description;
    this.serviceLocations = serviceLocations;
  }

}
